### PR TITLE
Restructure docs incorporating info from VEDA wiki

### PR DESCRIPTION
**Pull request by stephenkilbourn**
This is work in response to #169 

We have the [VEDA wiki](https://github.com/NASA-IMPACT/VEDA/wiki/) and these veda-docs. I've taken a pass at comparing [what info was different](https://docs.google.com/spreadsheets/d/1SUrY_pdoerN2ZYm-YnKzMcICo5ObMjQpvl01rxX7gt8/edit?usp=sharing) in the two documentation sources.

The main changes I made:
- Inserted a 2 column "table" section on the index page.
Screenshot 2024-09-11 at 12 35 36 PM

- Renamed "Services" to "Components" to match the wiki. I also added the Analytics and a Data Services page

- Updated the sidebar to match

TODO: add pages for missing links on new index page tables

### CODE REVIEW

**Review by j08lue — CHANGES_REQUESTED** [review]
Great to see the docs coming together!

I am wondering - why did you move everything under `/pages`?

Right now, the TOC and overall navigation is broken: when you now go to the root of the docs site (https://docs.openveda.cloud/pr-preview/pr-170), You get forwarded to the first thing in alphabetical order - the `pages/access.qmd` stub. Instead, https://docs.openveda.cloud/pr-preview/pr-170/pages takes you to the welcome page.

Do you have examples for the source tree structure of Quarto sites that you are modelling this after, @stephenkilbourn? 

Docs pages I am looking at for reference are those of other data platforms like ours: 

* https://github.com/microsoft/PlanetaryComputerExamples - not Quarto, though
* https://github.com/eu-cdse/documentation

Planetary Computer is particularly nice, because the root source tree is clean as a TOC in itself.

Which ever way we choose - please justify and make sure things work from a functional/navigation perspective. 🙏

**Comment by stephenkilbourn** [comment]
@j08lue 
> I am wondering - why did you move everything under `/pages`?
The logic was more from a developer experience perspective:  Especially as I started stubbing out the new pages, we had a plethora of `.qmd` files in the root (currently, 9 plus the index.qmd), along with directories for some sections. Having a logical grouping felt easier to locate and maintain.

**Comment by stephenkilbourn** [comment]
@j08lue - ok, I did a pass and links are all working in the preview deploy.  I can move page files to a flatter hierarchy if that's preferred. I focused on getting the sidebar updated for now.  It's now closer in style to the eu-csde layout, i think.
Screenshot 2024-09-13 at 4 33 21 PM

**Comment by stephenkilbourn** [comment]
These pages are currently "Coming Soon". I'm not finding source material for these. If someone is a SME on the topics, then I can work with them to incorporate details into the pages.
- title: "VEDA Access"
-- subtitle: "How to access the VEDA ecosystem"
- title: "VEDA Architecture"
-- subtitle: "VEDA Architecture"
- title: "VEDA Community and Support"
-- subtitle: "The VEDA Community"
- title: "How to Deploy VEDA"
-- subtitle: "How to Deploy Your Own VEDA"
- title: "VEDA Recommendations For Maintainers"
-- subtitle: "VEDA Recommendations For Maintainers"
- title: "Related Projects"
-- subtitle: "Related Projects"
- title: "VEDA Standards"
-- subtitle: "VEDA Standards"

**Comment by smohiudd** [comment]
- We currently have API documentation under `Components/APIs` and under `API Docs`. I was thinking API Docs would be either direct embed of the swagger docs ([like Planetary Computer](https://planetarycomputer.microsoft.com/docs/reference/stac/)) or link to swagger docs
- I don't think we'll be able to get through all the Todos in [your list](https://github.com/NASA-IMPACT/veda-docs/pull/170#issuecomment-2354130946) @stephenkilbourn in this PR. I'm not a big fan of "Coming Soon" pages. For this PR can we commit to including at least the "Software and Services Architecture" and "Repositories" sections to start and create issues for the other sections (and remove the stubs for now)? 
- For the "Getting Access" section, we can use this [existing text ](https://docs.openveda.cloud/services/jupyterhub.html#getting-access-to-vedas-jupyterhub-environment)from Jupyter Hub. I think that's the only service that requires auth.
- Can we bring back the sub list of pages for Contributing like the [current docs](https://docs.openveda.cloud/contributing/)?
- I agree with @j08lue about using the root source as the TOC without adding /pages. Is there a setting in Quarto where we can specify a root directory?

**Comment by stephenkilbourn** [comment]
The above 4 commits should address all 5 points.  I left the access section in the JupyterHub page, and just linked to that specific section from the TOC and sidebar.  My thought process is that once we get the veda-auth-central info, we can create a unique access page.

**Comment by smohiudd** [comment]
recommendation for architecture diagram: 

- include entire system diagram (everything in veda deploy: backend, airflow, auth, features api, grafana, etc.)
- high level boxes/icons, not necessarily details for each services (i.e. lambda, rds, vpc)

File: components/data-services.qmd [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+## Data Services
+
+The goal of VEDA Data Services is to support open science through community standard tooling, Application Programming Interfaces (APIs), and providing access to persistent, discoverable Earth science datasets. This approach includes:
+
+Hosting and making available key datasets in analysis-optimized formats
+Tying together VEDA products by providing access to hosted data, existing third-party geospatial datasets, and new datasets produced within the VEDA ecosystem
+Making data Findable, Accessible, Interoperable, and Reusable (FAIR), within and also beyond VEDA
+The Data Services hosts a suite of services that make these pointers possible.
```
Comment (ividito) [inline components/data-services.qmd:13]: These lines all render as one block for me, could we break it up with bullet points or other formatting?
```suggestion
- Hosting and making available key datasets in analysis-optimized formats
- Tying together VEDA products by providing access to hosted data, existing third-party geospatial datasets, and new datasets produced within the VEDA ecosystem
- Making data Findable, Accessible, Interoperable, and Reusable (FAIR), within and also beyond VEDA

The Data Services hosts a suite of services that make these pointers possible.
```
Screenshot 2024-10-08 at 15-17-43 Data Services – VEDA Documentation

**Comment by j08lue** [comment]
I am working on aligning the folder structure with the document hierarchy and fixing up menus and internal links, following this proposal

image

https://github.com/NASA-IMPACT/veda-architecture/issues/459#issuecomment-2333673003

Please avoid larger changes until I am done - and I hope you like the outcome. 😅

**Comment by smohiudd** [comment]
Thanks @j08lue! I think we are almost there with this PR after your hierarchy changes. As I mentioned last week, under "VEDA open-source ecosystem" I'm ok with just going with (1) Software and Services Architecture and (2) Repositories in this PR and then including the other sections in another PR as we work through assembling that information. Let me know what you think.

**Comment by j08lue** [comment]
I finished the document tree and hierarchy restructuring: https://docs.openveda.cloud/pr-preview/pr-170/

I made a lot of changes wrt the initial state of this PR and the organization in the wiki.

Left menu, repo folder structure, and home page overview hierarchy are close, but not 100% identical:
1. I kept the root `/notebooks` folder for backwards compatibility (not to break external links to our notebooks). 
2. Contributing docs are under `/nasa-veda-platform`, but linked at main level in the left menu.

I introduced a new section about `scientific computing` that hosts JupyterHub related docs now and can also have DPS docs etc. later.

All image and internal cross references should be working.

**Review by j08lue — APPROVED** [review]
The content needs more refinement, but I think it is good enough to be published and we should merge this to be able to make those edits incrementally on top of the changed structure.

**Comment by j08lue** [comment]
Good for merging now, IMO, @smohiudd.

**Review by anayeaye — APPROVED** [review]
This is great!

**Comment by smohiudd** [comment]
Thanks @jonas. I just pushed a couple changes [suggested](https://docs.google.com/document/d/1Ur5stWzJAd8kOcEUMCrei5aXkWxQEViz20Dh9pl38Xc/edit?tab=t.0) by @jerikac. We should good to merge this now!

**Review by smohiudd — APPROVED** [review]

**Pull request merged**